crowbar_framework/lib/crowbar/proposal_methods.rb

```diff
@@ -0,0 +1,145 @@
+module Crowbar
+  module ProposalMethods
```

Comment:
> Just so I understand: is this temporary until we get rid of ProposalObject, or do we plan to keep the public methods split forever?

This is primarily to keep the API's as close as possible in terms of
functionality.

When we get rid of ProposalObject, it'd be good to give this some more
thought as many of these methods are used only in the crowbar view and
could be moved to a 'decorator'. The rest would then go to the Proposal.